src/modal/index.tsx

```diff
@@ -81,6 +81,14 @@ const StyledModal = styled(Modal)`
     left: 8px;
     background-color: transparent;
   }
+
+  /* MediaSection is fullwith on small media. Counter the default padding with this */
```

Comment:
Can you add a line:
"Ideally, the Modal should not define any interal paddings and let the contained sections take care of the horizontal layout."